lib/endpoints/class-wp-rest-controller.php

```diff
@@ -11,121 +11,121 @@ public function register_routes() {
 	}
 
 	/**
-	 * Get a collection of items
+	 * Get a collection of items.
 	 *
 	 * @param WP_REST_Request $request Full data about the request.
-	 * @return WP_Error|WP_REST_Response
+	 * @return WP_Error|WP_REST_Response.
```

Comment:
@danielbachhuber  Though it does mention ` Use a period at the end.` 
What do you think about that? Let me know. I'll follow your lead.